lib/exec.js

```diff
@@ -1,9 +1,15 @@
-require('dotenv').config({silent: true});
-
 var spawn = require('child_process').spawn,
     objectAssign = require('object-assign');
 
 module.exports = function exec(script, program) {
+  var dotenvConfig = { silent: true };
+  ['encoding', 'path'].forEach(function (key) {
+    if (key in program) {
+      dotenvConfig[key] = program[key];
+    }
+  });
```

Comment:
can you simplify this by using [object-assign](https://www.npmjs.com/package/object-assign) (as we're [already](https://github.com/benoror/better-npm-run/blob/master/package.json#L32) using it [here](https://github.com/benoror/better-npm-run/blob/7f93c948ac3c4c1302716491d8c758158b5eafb9/lib/exec.js#L13)) 😄 

Something like:

```javascript
var dotenvConfig =  objectAssign({ silent: true }, program);
```
Or more specific:

```javascript
var dotenvConfig =  objectAssign({ silent: true }, {
  encoding: program.encoding,
  path: program.path
});
```

Thoughts?